src/Resources/views/Calls/list.html.twig

```diff
@@ -36,6 +36,7 @@
                             {{ macros.render_infos(call.info) }}
                             {{ macros.render_headers(call.request.headers, call.uri) }}
                             {{ macros.render_body(call.request.body) }}
+                            {{ macros.render_curl(call.curl) }}
```

Comment:
You should add an `if` statement, otherwise this won't work with older profiles.